src/main/java/com/treasuredata/client/model/TDApiKey.java

```diff
@@ -0,0 +1,32 @@
+package com.treasuredata.client.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.immutables.value.Value;
+
+@Value.Immutable
+@Value.Style(visibility = Value.Style.ImplementationVisibility.PACKAGE)
+@JsonSerialize(as = ImmutableTDApiKey.class)
+@JsonDeserialize(as = ImmutableTDApiKey.class)
+public abstract class TDApiKey
+{
+    @JsonProperty("key_type") public abstract String getKeyType();
+    @JsonProperty("account_id") public abstract Integer getAccountId();
+    @JsonProperty("user_id") public abstract Integer getUserId();
```

Comment:
I don't think we should expose account id and user id, because it's internal data of TD.

Reply:
See also `GET /v3/user/show` and `GET /v3/account/show`. We've exposed [user_id](https://github.com/treasure-data/td-client-java/blob/a44578d7442f03fec957144b2124a8023dccb31e/src/main/java/com/treasuredata/client/model/TDUser.java#L15) and [account_id](https://github.com/treasure-data/td-client-ruby/blob/d296734c1b2f79bae595fa4c1249c18bcb8c05a7/lib/td/client/api/account.rb#L16-L22) in public API clients. You may be interested in PLT-8361 on our JIRA too.